### PR TITLE
Post mailbox rename configuration

### PR DESCRIPTION
**Pull request by WilliamLoy**
Please do not merge until requested. I need to stage the article for our product team to review.

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c02b528c3a/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-57b100e5b8/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-a2c3af1472/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2f0c265173/how-to/). :bow:

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-f227b27ba9/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a806544623/how-to/)

**Comment by kmsholcomb** [comment]
Hi @DonNateR. As I mentioned in Slack, I think that this article could be slightly repurposed to provide an overview of all the things the user needs to know about renaming a mailbox. Even though it already provided the links to the procedures, the context was not quite clear (when to do what, etc.). So, I suggested a bit of rewording in the title and intro and a slight reorg of the information. Let me know if you think this works. If so, then I would also recommend that all of the related articles provide a link back to this article, so that everything is connected. Please let me know if you have any questions. Thanks!

[Rename Rackspace Email address setup FAQ_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1293540/Rename.Rackspace.Email.address.setup.FAQ_editsKH.docx)
[Rename Rackspace Email address setup FAQ_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1293541/Rename.Rackspace.Email.address.setup.FAQ_editsKH.pdf)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-3192cb0094/how-to/)

File: content/rackspace-email/renaming-a-rackspace-email-mailbox-overview.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+### Troubleshooting
+
+#### I renamed my Rackspace Email address, but now Outlook and my phone are not receiving messages. How do I fix this?
+
+Your devices are not receiving mail because they are configured to connect using the old email address. If you log in at [apps.rackspace.com](apps.rackspace.com) using the new email address, you can see that the mailbox is still receiving mail. Select your mail client in [Desktop clients](#desktop-clients) or [Mobile clients](#mobile-clients) for specific instructions.
+
+#### I followed the configuration instructions and now I am missing mail! What happened?
+
+As we recommended you should back up local data before making changes. In many cases this issue is easily resolved. If you are missing mail data, it might be that your old configuration was storing data locally to your device. If backed up the data, you can import it into the new configuration.
```
Comment (catlook) [inline content/rackspace-email/renaming-a-rackspace-email-mailbox-overview.md]: [I'd remove the first sentence and add the following mention]
In many cases, this issue ...
... If you backed up the data, as we recommended, you can ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-913229f4c1/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c7575775ee/how-to/)

**Comment by WilliamLoy** [comment]
Before this is merged we need to discuss the use of "rename email address" as opposed to "renamed mailbox". Our product team believe "rename email address" is a more accurate representation of the process and it matches what is written in the control panel.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8ffba98c5c/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-7ae9fcd342/how-to/)

**Pull request merged**